### PR TITLE
Adds Degree Date and Committee Members to ETD.

### LINKED ISSUE #1237: Add Degree and Degree Year fields to ETD works
Scholar 2.x has required `Degree` and `Degree Year` fields on the ETD work form (look just under college and departments).

These fields need to be added to the ETDs edit form in Scholar 3.

### PR DESCRIPTION
**Pull request by scherztc**
Fixes #1237 

Added two etd specific metadata fields.  degree_date and committee_member.

### CODE REVIEW

**Comment by hortongn** [comment]
@scherztc You have a file conflict in your PR

File: spec/models/etd_spec.rb [hunk 1]
```diff
@@ -98,6 +98,7 @@
     it { should respond_to(:description) }
     it { should respond_to(:publisher) }
     it { should respond_to(:degree) }
+    it { should respond_to(:degree_date) }
```
Comment (hortongn) [inline spec/models/etd_spec.rb]: @scherztc Should this test also verify that the model responds to :committee_member ?

**Comment by hortongn** [comment]
@scherztc The `Degree Year`  field needs to use a JS date picker like the one added in Curate here https://github.com/uclibs/curate/commit/fe6af05f89b11ca29880e154c8cac7423c822dbf

Also be sure to call the field `Degree Year` instead of `Degree Date`

For the `Committee Member` field include the specific instructions from Scholar 2: "Enter any additional members of the thesis/dissertation committee. Please enter the name in LastName, FirstName order."

**Comment by scherztc** [comment]
@hortongn .  I updated the help description quickly for committee_members.  The javascript for date validation was not that simple.  We have an issue for date validations through out the forms and I have included degree_date to that list of validations. https://github.com/uclibs/scholar_uc/issues/831
I am requesting that this PR be merged and we deal with data validations all together.

**Comment by hortongn** [comment]
If we merge this without the year picker, we should probably update the helper text for that field to make it clear to the user what data format Scholar expects.

**Comment by jamesvanmil** [comment]
In Scholar 2 I think that degree date was just date created with a different label - we should probably keep it the same way

**Comment by scherztc** [comment]
Okay.  I have changed the label for date_created on ETDs, modified the help text, and added committee_members.  The use of date_created instead of a new field also supports our point that date validation is a different issue.

**Pull request merged**